### PR TITLE
FIX/TST: DeltaListScan should accept a list, not only an ndarray.

### PR DESCRIPTION
**Pull request by danielballan**
Straightforward fix to one issue in #225

### CODE REVIEW

**Comment by danielballan** [comment]
cc @vivekthampy -- this is one of the problems I worked around when we built a higher-dimensional scan

File: bluesky/scans.py [hunk 1]
```diff
@@ -226,7 +226,7 @@ def _pre_scan(self):
             raise NotImplementedError("Can't DScan this motor")
         key, = ret.keys()
         self.init_pos = ret[key]['value']
-        self._abs_steps = self.steps + self.init_pos
+        self._abs_steps = np.asarray(self.steps) + self.init_pos
```
Comment (ericdill) [inline bluesky/scans.py]: is `self.init_pos` guaranteed to be an int/float?

Comment (ericdill) [inline bluesky/scans.py]: Ok, dan says "yes".

**Comment by ericdill** [comment]
LGTM

**Pull request merged**